redfish-core/lib/processor.hpp

```diff
@@ -1420,4 +1831,110 @@ inline void requestRoutesProcessor(App& app)
             });
 }
 
+inline void requestRoutesSubProcessors(App& app)
+{
+    BMCWEB_ROUTE(app,
+                 "/redfish/v1/Systems/system/Processors/<str>/SubProcessors")
+        .privileges(redfish::privileges::getProcessor)
```

Comment:
This should be getProcessorCollection

Reply:
Done - Refreshed the https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/38570/43 patch set and the comment addressed in that patch set.